lib/chef-dk/command/generator_commands.rb

```diff
@@ -69,6 +69,12 @@ module SharedGeneratorOptions
         :description => "Use to set arbitrary attribute KEY to VALUE in the code_generator cookbook",
         :default => [],
         :proc => Proc.new { |s| generator_args <<  s.split("=").map(&:strip) }
+
+      option :pipeline,
+        :short => "-f PIPELINE",
```

Comment:
Would introduce less ambiguity if we just called this `branch` and had the description state that this determines which branch to target?